htlcswitch/mailbox.go

```diff
@@ -454,9 +561,16 @@ type mailOrchestrator struct {
 	unclaimedPackets map[lnwire.ShortChannelID][]*htlcPacket
 }
 
+type mailOrchConfig struct {
+	forwardPackets func(chan struct{}, ...*htlcPacket) chan error
```

Comment:
Missing `godoc` comments, can really just be copied over from the `mailBoxConfig`. Or if we want to cut down on the redundant attributes, various forms of struct embedding could help here.